smo-frontend/src/assets/translations.json

```diff
@@ -427,5 +427,148 @@
+          "Label": "Bir sonraki istasyon bilgisini göster"
+        },
+        "MapTheme": "Harita Teması (Karanlık Mod İçin)",
+        "alternativeTheme": {
+          "Label": "Monokrom harita"
+        }
+      },
+      "TrainMakerPopup": {
+        "NextSignal": "Bir Sonraki Sinyal:",
+        "Consist": "Tren Yapısı (Motor ve Vagonlar)",
+        "OOB": "Oynanabilir bölgenin dışında",
+        "Unfollow": "Takibi Bırak",
+        "Follow": "Takip Et",
+        "Unpin": "Sabitlemeyi Kaldır",
+        "Pin": "Sabitle",
+        "ShowRoute": "Rotayı Göster",
+        "HideRoute": "Rotayı Gizle"
+      },
+      "SelectedRoute": "Seçili Rota",
+      "Hide": "Gizle",
+      "SignalMarker": {
+        "SignalSpeed": "Sinyal hızı: ",
+        "TrainApproaching": "Tren <0>{{TrainNoLocal}} ({{TrainName}})</0> bu sinyale <1>{{Velocity}} km/s</1> hızla yaklaşıyor ve <2>{{DistanceToSignalInFront}} m</2> uzaklıkta.",
+        "TrainAhead": "Tren <0>{{TrainNoLocal}} ({{TrainName}})</0> <1>bir sonraki blokta</1> bulunuyor ve <2>{{Velocity}} km/s</2> hızla ilerliyor. <3>{{DistanceToSignalInFront}} m</3> uzaklıkta ve <4>{{SignalName}}</4> sinyaline yaklaşıyor.",
+        "NextSignalWithTrainAhead": "Bir sonraki sinyalin <0>{{nextSignalWithTrainAhead}}</0> <1>önündeki blokta</1> bir tren var.",
+        "NoTrainApproaching": {
+          "Title": "Yaklaşan tren yok.",
+          "Description": "Yaklaşan bir tren olmadan sinyaller hakkında bilgi gösteremeyiz."
+        },
+        "Extra": "Ekstra:",
+        "Type": "Tür:",
+        "Role": "Rol:",
+        "Accuracy": "Doğruluk:",
+        "Unknown": "Bilinmiyor",
+        "PreviousSignals": "Önceki sinyaller",
+        "NextSignals": "Sonraki sinyaller",
+        "HideSignalLines": "Sinyal çizgilerini gizle",
+        "ShowSignalLines": "Sinyal çizgilerini göster",
+        "ShowSignalLinesFurther": {
+          "Description": "Daha fazla sinyal çizgisi göster",
+          "Title": "Sinyal sayısına bağlı olarak bu yavaş olabilir"
+        },
+        "Red": "Kırmızı",
+        "Orange": "Turuncu",
+        "Blue": "Mavi",
+        "Purple": "Mor",
+        "AdminActions": {
+          "Title": "Yönetici işlemleri:",
+          "DeleteSignal": "Bu sinyali sil",
+          "UnFinalizePrev": "Önceki sinyallerin onayını kaldır",
+          "FinalizePrev": "Önceki sinyalleri onayla",
+          "UnFinalizeNext": "Sonraki sinyallerin onayını kaldır",
+          "FinalizeNext": "Sonraki sinyalleri onayla"
+        },
+        "Footer": {
+          "Finalized": "(Bu veri kontrol edildi ve doğrudur)",
+          "UnFinalized": "(Bu veri kontrol edilmedi ve yanlış olabilir)"
+        }
+      },
+      "StationDisplay": {
+        "StopType": {
+          "CommercialStop": "Yolcu istasyonu",
+          "NoncommercialStop": "Bekleme istasyonu"
+        },
+        "ShouldArrive": {
+          "Now": "Bu tren şu anda bu istasyona varmalı.",
+          "Past": "Bu tren bu istasyona {{time}} önce varmalıydı.",
```

Comment:
"önce" will get duplicated here, because it's already there by `moment`

Reply:
Fix has been applied into commit 8f8c4cfcc3852b8f43b40b523334e1f0f233d828